### PR TITLE
Initial commit from template

### PR DESCRIPTION
**Pull request by jeduden**

### CODE REVIEW

File: docs/installation.rst [hunk 1]
```diff
@@ -0,0 +1,7 @@
+============
+Installation
+============
+
+At the command line::
+
+    $ pip install py-detox-bridge
```
Comment (sergray) [inline docs/installation.rst]: This should be detox-bridge

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+                 'Operating System :: OS Independent',
+                 'Programming Language :: Python',
+                 'Programming Language :: Python :: 2',
+                 'Programming Language :: Python :: 2.7',
+                 'Topic :: Internet :: WWW/HTTP'],
+ 'description': 'A python bridge to the detox greybox testing library',
+ 'include_package_data': True,
+ 'install_requires': [],
+ 'long_description': 'detox_bridge\n===========================\n\n.. image:: https://secure.travis-ci.org/kpn-digital/detox_bridge.svg?branch=master\n    :target:  http://travis-ci.org/kpn-digital/detox_bridge?branch=master\n\n.. image:: https://img.shields.io/codecov/c/github/kpn-digital/detox_bridge/master.svg\n    :target: http://codecov.io/github/kpn-digital/detox_bridge?branch=master\n\n.. image:: https://img.shields.io/pypi/v/detox_bridge.svg\n    :target: https://pypi.python.org/pypi/detox_bridge\n\n.. image:: https://readthedocs.org/projects/detox_bridge/badge/?version=latest\n    :target: http://detox_bridge.readthedocs.org/en/latest/?badge=latest\n',
+ 'name': 'py-detox-bridge',
```
Comment (sergray) [inline setup.py:17]: Please omit the py- prefix of repository from the package name

**Review by sergray — CHANGES_REQUESTED** [review]
Python package name should not have a `py-` prefix

**Comment by jeduden** [comment]
@sergray fixed reported issues.

**Review by sergray — APPROVED** [review]

**Pull request merged**